app/webpacker/components/Requests/Errored.js

```diff
@@ -3,21 +3,25 @@ import React from 'react';
 import { Message, Icon } from 'semantic-ui-react';
 
 function Errored({
-  componentName,
+  componentName, error,
 }) {
   return (
     <Message icon negative>
       <Icon name="warning sign" />
       <Message.Content>
         <Message.Header>Oh no :(</Message.Header>
-        Something went wrong while loading the data
-        {componentName && (
-        <>
-          {' '}
-          for the component &apos;
-          {componentName}
-          &apos;
-        </>
+        {String(error) || (
+          <>
+            Something went wrong while loading the data
+            {componentName && (
+            <>
+              {' '}
+              for the component &apos;
+              {componentName}
+              &apos;
+            </>
+            )}
+          </>
```

Comment:
I don't think the error message alone is always clear enough. Remember that this is a pretty generic component that's being used all over the place, and error messages are not always self-contained.
So perhaps stick with the existing text (pre your change) and _append_ the error text additionally, if present?